pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -115,6 +115,7 @@ public class ClientCnx extends PulsarHandler {
                     .expectedItems(16)
                     .concurrencyLevel(1)
                     .build();
+    @Getter
```

Comment:
It's better to return an immutable map to prevent anyone to modify it externally.

Reply:
Done.